package.json

```diff
@@ -19,7 +25,7 @@
     "template",
     "eslint"
   ],
-  "author": "Israel Laguan",
+  "author": "Israel Laguan, Fabricio Garcia",
```

Comment:
You might want to use the `contributors` field instead of the `author` field to list more than one: https://docs.npmjs.com/files/package.json#people-fields-author-contributors

Reply:
OK, we can change that!